tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1422,3 +1422,19 @@ Events:
 		Event parameters:	<none>
 
 		This event indicate that node was provisioned.
+
+	Opcode 0x84 - Link open Event
+
+		Controller Index:	<controller id>
+		Event parameters:	Bearer (1 octet)
```

Comment:
You're missing the possible values for Bearer

Reply:
Yes, indeed

Reply:
done